composer.json

```diff
@@ -20,14 +20,12 @@
   ],
   "require": {
     "silverstripe/framework": "^4@dev",
+    "intervention/image": "^2.3",
     "php": ">=5.6.0"
   },
   "require-dev": {
     "silverstripe/versioned": "^1@dev",
-    "phpunit/PHPUnit": "~4.8"
-  },
-  "suggest": {
-    "ext-gd": "*"
+    "phpunit/phpunit": "^5.7"
```

Comment:
This "fixes" the master build because it raises the PHPUnit dep from 4.x to 5.x, which in turn seems to fix a weird bug around invoking @dataProvider functions before calling test setup (which results in No config manifests available)